src/xdp/xdp_user_monitoring_prog/Makefile

```diff
@@ -0,0 +1,9 @@
+# SPDX-License-Identifier: (GPL-2.0 OR BSD-2-Clause)
```

Comment:
Can you change this file to module.mk so we can just compile everything from the root directory?

Reply:
The xdp_user_monitoring_prog relies on the library support from the xdp_tutorial (https://github.com/xdp-project/xdp-tutorial). Curretnly, I compile the xdp_user_monitoring_prog under the directory of that. I'm not sure how to handle this, could you give me some suggestions?

Reply:
Can we compile it under the directory of xdp_tutorial and then move it to "mizar/tools/"?